src/main/java/ch/njol/skript/expressions/ExprExperience.java

```diff
@@ -124,13 +129,15 @@ else if (e instanceof BlockBreakEvent)
 				}
 			}
 		else
-			d = 0;
-		
-		d = Math.max(0, Math.round(d));
-		if (e instanceof ExperienceSpawnEvent)
-			((ExperienceSpawnEvent) e).setSpawnedXP((int) d);
+			experience = 0;
+
+		experience = Math.max(0, Math.round(experience));
```

Comment:
You can't continue to have this round method. The round method originally existed for the double, which is now an integer from the looks of it. Math.round(integer) doesn't exist, it only does double and floats, so no clue how this is valid code unless it's somehow converting it to a double/float.